tests/integration/common/s2n_test_openssl.py

```diff
@@ -77,6 +77,6 @@ def get_openssl(scenario):
     return openssl
```

Comment:
We can (always use / force through peer_flags) the `-noservername` parameter to get rid of that Can't use SSL_get_servername error.